source/Cosmos.System2/Graphics/Fonts/ASC16.cs

```diff
@@ -0,0 +1,39 @@
+        static MemoryStream ASC16FontMS = new MemoryStream(Convert.FromBase64String(ASC16Base64));
+
+        public static void _DrawACSIIString(this DoubleBufferedVMWareSVGAII vMWareSVGAII, string s, uint color, uint x, uint y)
```

Comment:
You shouldn't use a new function to draw a string, we have one in Canvas.cs
What do you think @quajak?

Reply:
Yeah, unless this has some advantage vs the one we have in canvas, I dont see the need.

Reply:
changed